2015-11-12 17:30:47,414 - octoprint.plugin.core - INFO - 3 plugin(s) registered with the system:
|  CuraEngine (bundled) = /home/pi/OctoPrint/src/octoprint/plugins/cura
|  Discovery (bundled) = /home/pi/OctoPrint/src/octoprint/plugins/discovery
|  Netconnectd Client (bundled) = /home/pi/OctoPrint/src/octoprint/plugins/octoprint_netconnectd
2015-11-12 17:30:47,737 - octoprint.server - INFO - Listening on http://0.0.0.0:5000
2015-11-12 17:30:47,741 - octoprint.plugins.discovery - INFO - Registered OctoPrint instance on octopi for SSDP
2015-11-12 17:31:21,064 - octoprint.plugins.cura - INFO - Path to CuraEngine has not been configured yet or does not exist (currently set to None), Cura will not be selectable for slicing
2015-11-12 17:31:23,920 - octoprint.server.util.sockjs - INFO - New connection from client: 192.168.0.199
2015-11-12 17:32:43,435 - octoprint.server.util.sockjs - INFO - Client connection closed: 192.168.0.199
2015-11-12 17:32:44,500 - octoprint.plugins.cura - INFO - Path to CuraEngine has not been configured yet or does not exist (currently set to None), Cura will not be selectable for slicing
2015-11-12 17:32:46,754 - octoprint.server.util.sockjs - INFO - New connection from client: 192.168.0.199
2015-11-12 17:43:54,068 - octoprint.plugins.cura - INFO - Path to CuraEngine has not been configured yet or does not exist (currently set to None), Cura will not be selectable for slicing
2015-11-12 17:43:54,519 - octoprint.server.util.sockjs - INFO - New connection from client: 192.168.0.199
2015-11-12 17:46:09,449 - octoprint.server.api - INFO - Performing command: sudo shutdown -h now
2015-11-12 17:46:37,244 - octoprint.server - INFO - Starting OctoPrint 1.2.0-dev-705-ga34cbc5-dirty (devel branch)
2015-11-12 17:46:37,249 - octoprint.plugin.core - INFO - Loading plugins from /home/pi/.octoprint/plugins, /home/pi/OctoPrint/src/octoprint/plugins and installed plugin packages...
2015-11-12 17:46:37,453 - octoprint.plugins.discovery - INFO - pybonjour is not installed, Zeroconf Discovery won't be available
2015-11-12 17:46:37,456 - octoprint.plugin.core - INFO - Found 3 plugin(s) providing 2 mixin implementations, 0 hook handlers
2015-11-12 17:46:37,464 - octoprint.filemanager - INFO - Adding backlog items from local to analysis queue
2015-11-12 17:46:39,028 - octoprint.plugin.core - INFO - 3 plugin(s) registered with the system:
|  CuraEngine (bundled) = /home/pi/OctoPrint/src/octoprint/plugins/cura
|  Discovery (bundled) = /home/pi/OctoPrint/src/octoprint/plugins/discovery
|  Netconnectd Client (bundled) = /home/pi/OctoPrint/src/octoprint/plugins/octoprint_netconnectd
2015-11-12 17:46:39,344 - octoprint.server - INFO - Listening on http://0.0.0.0:5000
2015-11-12 17:46:39,345 - octoprint.plugins.discovery - INFO - Registered OctoPrint instance on octopi for SSDP
2015-11-12 17:47:52,515 - octoprint.plugins.cura - INFO - Path to CuraEngine has not been configured yet or does not exist (currently set to None), Cura will not be selectable for slicing
2015-11-12 17:47:52,973 - octoprint.server.util.sockjs - INFO - New connection from client: 192.168.0.199
2015-11-12 18:19:50,551 - octoprint.server - INFO - Starting OctoPrint 1.2.0-dev-705-ga34cbc5-dirty (devel branch)
2015-11-12 18:19:50,556 - octoprint.plugin.core - INFO - Loading plugins from /home/pi/.octoprint/plugins, /home/pi/OctoPrint/src/octoprint/plugins and installed plugin packages...
2015-11-12 18:19:50,764 - octoprint.plugins.discovery - INFO - pybonjour is not installed, Zeroconf Discovery won't be available
2015-11-12 18:19:50,767 - octoprint.plugin.core - INFO - Found 3 plugin(s) providing 2 mixin implementations, 0 hook handlers
2015-11-12 18:19:50,774 - octoprint.filemanager - INFO - Adding backlog items from local to analysis queue
2015-11-12 18:19:52,491 - octoprint.plugin.core - INFO - 3 plugin(s) registered with the system:
|  CuraEngine (bundled) = /home/pi/OctoPrint/src/octoprint/plugins/cura
|  Discovery (bundled) = /home/pi/OctoPrint/src/octoprint/plugins/discovery
|  Netconnectd Client (bundled) = /home/pi/OctoPrint/src/octoprint/plugins/octoprint_netconnectd
2015-11-12 18:19:52,807 - octoprint.plugins.discovery - INFO - Registered OctoPrint instance on octopi for SSDP
2015-11-12 18:19:52,809 - octoprint.server - INFO - Listening on http://0.0.0.0:5000
2015-11-12 18:20:16,636 - octoprint.plugins.cura - INFO - Path to CuraEngine has not been configured yet or does not exist (currently set to None), Cura will not be selectable for slicing
2015-11-12 18:20:19,421 - octoprint.server.util.sockjs - INFO - New connection from client: 192.168.0.199
2015-11-12 18:32:44,611 - octoprint.server.util.sockjs - INFO - Client connection closed: 192.168.0.199
2015-11-12 18:32:47,913 - octoprint.plugins.cura - INFO - Path to CuraEngine has not been configured yet or does not exist (currently set to None), Cura will not be selectable for slicing
2015-11-12 18:32:50,733 - octoprint.server.util.sockjs - INFO - New connection from client: 192.168.0.199
2015-11-12 18:37:02,857 - octoprint.server.api - INFO - Performing command: sudo shutdown -h now
2015-11-12 18:37:29,101 - octoprint.server - INFO - Starting OctoPrint 1.2.0-dev-705-ga34cbc5-dirty (devel branch)
2015-11-12 18:37:29,104 - octoprint.plugin.core - INFO - Loading plugins from /home/pi/.octoprint/plugins, /home/pi/OctoPrint/src/octoprint/plugins and installed plugin packages...
2015-11-12 18:37:29,241 - octoprint.plugins.discovery - INFO - pybonjour is not installed, Zeroconf Discovery won't be available
2015-11-12 18:37:29,242 - octoprint.plugin.core - INFO - Found 3 plugin(s) providing 2 mixin implementations, 0 hook handlers
2015-11-12 18:37:29,247 - octoprint.filemanager - INFO - Adding backlog items from local to analysis queue
2015-11-12 18:37:30,666 - octoprint.plugin.core - INFO - 3 plugin(s) registered with the system:
|  CuraEngine (bundled) = /home/pi/OctoPrint/src/octoprint/plugins/cura
|  Discovery (bundled) = /home/pi/OctoPrint/src/octoprint/plugins/discovery
|  Netconnectd Client (bundled) = /home/pi/OctoPrint/src/octoprint/plugins/octoprint_netconnectd
2015-11-12 18:37:30,982 - octoprint.plugins.discovery - INFO - Registered OctoPrint instance on octopi for SSDP
2015-11-12 18:37:30,986 - octoprint.server - INFO - Listening on http://0.0.0.0:5000
2015-11-13 02:02:37,184 - octoprint.plugins.cura - INFO - Path to CuraEngine has not been configured yet or does not exist (currently set to None), Cura will not be selectable for slicing
2015-11-13 02:02:39,978 - octoprint.server.util.sockjs - INFO - New connection from client: 192.168.0.199
2015-11-13 02:06:22,912 - octoprint.server.util.sockjs - INFO - Client connection closed: 192.168.0.199
2015-11-13 02:06:36,042 - octoprint.plugins.cura - INFO - Path to CuraEngine has not been configured yet or does not exist (currently set to None), Cura will not be selectable for slicing
2015-11-13 02:06:37,150 - octoprint.server.util.sockjs - INFO - New connection from client: 192.168.0.199
2015-11-13 06:04:30,584 - octoprint.server.util.sockjs - INFO - Client connection closed: 192.168.0.199
2015-11-13 06:04:35,811 - octoprint.server.util.sockjs - INFO - New connection from client: 192.168.0.199
2015-11-13 09:00:00,032 - octoprint.server.util.sockjs - INFO - Client connection closed: 192.168.0.199
2015-11-13 09:00:15,846 - octoprint.plugins.cura - INFO - Path to CuraEngine has not been configured yet or does not exist (currently set to None), Cura will not be selectable for slicing
2015-11-13 09:00:18,604 - octoprint.server.util.sockjs - INFO - New connection from client: 192.168.0.199
2015-11-13 09:00:51,044 - octoprint.server.util.sockjs - INFO - Client connection closed: 192.168.0.199
2015-11-13 10:00:19,886 - octoprint.plugins.cura - INFO - Path to CuraEngine has not been configured yet or does not exist (currently set to None), Cura will not be selectable for slicing
2015-11-13 10:00:22,674 - octoprint.server.util.sockjs - INFO - New connection from client: 192.168.0.199
2015-11-13 10:00:44,186 - octoprint.server.util.sockjs - INFO - Client connection closed: 192.168.0.199
2015-11-13 10:48:05,294 - octoprint.plugins.cura - INFO - Path to CuraEngine has not been configured yet or does not exist (currently set to None), Cura will not be selectable for slicing
2015-11-13 10:48:06,367 - octoprint.server.util.sockjs - INFO - New connection from client: 192.168.0.199
2015-11-13 10:49:05,526 - octoprint.server.util.sockjs - INFO - Client connection closed: 192.168.0.199
2015-11-13 12:48:52,781 - octoprint.plugins.cura - INFO - Path to CuraEngine has not been configured yet or does not exist (currently set to None), Cura will not be selectable for slicing
2015-11-13 12:48:55,678 - octoprint.server.util.sockjs - INFO - New connection from client: 192.168.0.199
2015-11-13 13:54:42,038 - octoprint.plugins.cura - INFO - Path to CuraEngine has not been configured yet or does not exist (currently set to None), Cura will not be selectable for slicing
2015-11-13 13:54:42,400 - octoprint.server.util.sockjs - INFO - New connection from client: 192.168.0.194
2015-11-13 14:52:56,719 - octoprint.server.api - INFO - Performing command: sudo shutdown -h now
2015-11-13 14:53:25,314 - octoprint.server - INFO - Starting OctoPrint 1.2.0-dev-705-ga34cbc5-dirty (devel branch)
2015-11-13 14:53:25,318 - octoprint.plugin.core - INFO - Loading plugins from /home/pi/.octoprint/plugins, /home/pi/OctoPrint/src/octoprint/plugins and installed plugin packages...
2015-11-13 14:53:25,449 - octoprint.plugins.discovery - INFO - pybonjour is not installed, Zeroconf Discovery won't be available
2015-11-13 14:53:25,451 - octoprint.plugin.core - INFO - Found 2 plugin(s) providing 2 mixin implementations, 0 hook handlers
2015-11-13 14:53:25,455 - octoprint.filemanager - INFO - Adding backlog items from local to analysis queue
2015-11-13 14:53:26,563 - octoprint.plugin.core - INFO - 2 plugin(s) registered with the system:
|  CuraEngine (bundled) = /home/pi/OctoPrint/src/octoprint/plugins/cura
|  Discovery (bundled) = /home/pi/OctoPrint/src/octoprint/plugins/discovery
2015-11-13 14:53:26,768 - octoprint.server - INFO - Listening on http://0.0.0.0:5000
2015-11-13 14:53:26,770 - octoprint.plugins.discovery - INFO - Registered OctoPrint instance "Kudo3D" for SSDP
2015-11-13 14:53:32,447 - octoprint.plugins.cura - INFO - Path to CuraEngine has not been configured yet or does not exist (currently set to None), Cura will not be selectable for slicing
2015-11-13 14:53:32,939 - octoprint.server.util.sockjs - INFO - New connection from client: 192.168.0.194
2015-11-13 16:25:07,370 - octoprint.server.api - INFO - Performing command: sudo shutdown -h now
2015-11-13 16:25:29,680 - octoprint.server - INFO - Starting OctoPrint 1.2.0-dev-705-ga34cbc5-dirty (devel branch)
2015-11-13 16:25:29,683 - octoprint.plugin.core - INFO - Loading plugins from /home/pi/.octoprint/plugins, /home/pi/OctoPrint/src/octoprint/plugins and installed plugin packages...
2015-11-13 16:25:29,815 - octoprint.plugins.discovery - INFO - pybonjour is not installed, Zeroconf Discovery won't be available
2015-11-13 16:25:29,816 - octoprint.plugin.core - INFO - Found 2 plugin(s) providing 2 mixin implementations, 0 hook handlers
2015-11-13 16:25:29,821 - octoprint.filemanager - INFO - Adding backlog items from local to analysis queue
2015-11-13 16:25:30,944 - octoprint.plugin.core - INFO - 2 plugin(s) registered with the system:
|  CuraEngine (bundled) = /home/pi/OctoPrint/src/octoprint/plugins/cura
|  Discovery (bundled) = /home/pi/OctoPrint/src/octoprint/plugins/discovery
2015-11-13 16:25:31,148 - octoprint.server - INFO - Listening on http://0.0.0.0:5000
2015-11-13 16:25:31,150 - octoprint.plugins.discovery - INFO - Registered OctoPrint instance "Kudo3D" for SSDP
2015-11-13 16:25:31,375 - octoprint.server.util.sockjs - INFO - New connection from client: 192.168.0.194
2015-11-13 16:26:52,807 - octoprint.server.api - INFO - Performing command: sudo shutdown -r now
2015-11-13 16:27:14,815 - octoprint.server - INFO - Starting OctoPrint 1.2.0-dev-705-ga34cbc5-dirty (devel branch)
2015-11-13 16:27:14,819 - octoprint.plugin.core - INFO - Loading plugins from /home/pi/.octoprint/plugins, /home/pi/OctoPrint/src/octoprint/plugins and installed plugin packages...
2015-11-13 16:27:14,951 - octoprint.plugins.discovery - INFO - pybonjour is not installed, Zeroconf Discovery won't be available
2015-11-13 16:27:14,953 - octoprint.plugin.core - INFO - Found 2 plugin(s) providing 2 mixin implementations, 0 hook handlers
2015-11-13 16:27:14,958 - octoprint.filemanager - INFO - Adding backlog items from local to analysis queue
2015-11-13 16:27:15,446 - octoprint.plugin.core - INFO - 2 plugin(s) registered with the system:
|  CuraEngine (bundled) = /home/pi/OctoPrint/src/octoprint/plugins/cura
|  Discovery (bundled) = /home/pi/OctoPrint/src/octoprint/plugins/discovery
2015-11-13 16:27:15,651 - octoprint.server - INFO - Listening on http://0.0.0.0:5000
2015-11-13 16:27:15,653 - octoprint.plugins.discovery - INFO - Registered OctoPrint instance "Kudo3D" for SSDP
2015-11-13 16:27:15,767 - octoprint.server.util.sockjs - INFO - New connection from client: 192.168.0.194
2015-11-13 16:27:59,124 - octoprint.server.api - INFO - Performing command: sudo service octoprint restart
2015-11-13 16:28:17,153 - octoprint.server.api - INFO - Performing command: sudo service octoprint restart
2015-11-13 16:32:23,461 - octoprint.server.api - INFO - Performing command: sudo shutdown -h now
2015-11-13 16:32:49,238 - octoprint.server - INFO - Starting OctoPrint 1.2.0-dev-705-ga34cbc5-dirty (devel branch)
2015-11-13 16:32:49,242 - octoprint.plugin.core - INFO - Loading plugins from /home/pi/.octoprint/plugins, /home/pi/OctoPrint/src/octoprint/plugins and installed plugin packages...
2015-11-13 16:32:49,434 - octoprint.plugins.discovery - INFO - pybonjour is not installed, Zeroconf Discovery won't be available
2015-11-13 16:32:49,437 - octoprint.plugin.core - INFO - Found 2 plugin(s) providing 2 mixin implementations, 0 hook handlers
2015-11-13 16:32:49,444 - octoprint.filemanager - INFO - Adding backlog items from local to analysis queue
2015-11-13 16:32:50,210 - octoprint.plugin.core - INFO - 2 plugin(s) registered with the system:
|  CuraEngine (bundled) = /home/pi/OctoPrint/src/octoprint/plugins/cura
|  Discovery (bundled) = /home/pi/OctoPrint/src/octoprint/plugins/discovery
2015-11-13 16:32:50,525 - octoprint.server - INFO - Listening on http://0.0.0.0:5000
2015-11-13 16:48:59,529 - octoprint.plugins.cura - INFO - Path to CuraEngine has not been configured yet or does not exist (currently set to None), Cura will not be selectable for slicing
2015-11-13 16:48:59,903 - octoprint.server.util.sockjs - INFO - New connection from client: 192.168.0.194
2015-11-13 17:00:52,033 - octoprint.server.util.sockjs - INFO - Client connection closed: 192.168.0.194
2015-11-13 17:08:41,222 - octoprint.plugins.cura - INFO - Path to CuraEngine has not been configured yet or does not exist (currently set to None), Cura will not be selectable for slicing
2015-11-13 17:08:41,532 - octoprint.server.util.sockjs - INFO - New connection from client: 192.168.0.194
2015-11-13 17:15:15,468 - octoprint.server.util.sockjs - INFO - Client connection closed: 192.168.0.194
2015-11-13 17:15:16,554 - octoprint.plugins.cura - INFO - Path to CuraEngine has not been configured yet or does not exist (currently set to None), Cura will not be selectable for slicing
2015-11-13 17:15:16,872 - octoprint.server.util.sockjs - INFO - New connection from client: 192.168.0.194
2015-11-13 17:22:45,941 - octoprint.server.api - INFO - Performing command: sudo shutdown -r now
2015-11-13 17:23:10,371 - octoprint.server - INFO - Starting OctoPrint 1.2.0-dev-705-ga34cbc5-dirty (devel branch)
2015-11-13 17:23:10,375 - octoprint.plugin.core - INFO - Loading plugins from /home/pi/.octoprint/plugins, /home/pi/OctoPrint/src/octoprint/plugins and installed plugin packages...
2015-11-13 17:23:10,568 - octoprint.plugins.discovery - INFO - pybonjour is not installed, Zeroconf Discovery won't be available
2015-11-13 17:23:10,571 - octoprint.plugin.core - INFO - Found 2 plugin(s) providing 2 mixin implementations, 0 hook handlers
2015-11-13 17:23:10,578 - octoprint.filemanager - INFO - Adding backlog items from local to analysis queue
2015-11-13 17:23:12,240 - octoprint.plugin.core - INFO - 2 plugin(s) registered with the system:
|  CuraEngine (bundled) = /home/pi/OctoPrint/src/octoprint/plugins/cura
|  Discovery (bundled) = /home/pi/OctoPrint/src/octoprint/plugins/discovery
2015-11-13 17:23:12,551 - octoprint.server - INFO - Listening on http://0.0.0.0:5000
2015-11-13 17:23:12,552 - octoprint.plugins.discovery - INFO - Registered OctoPrint instance "Kudo3D" for SSDP
2015-11-13 17:23:17,018 - octoprint.server.util.sockjs - INFO - New connection from client: 192.168.0.194
2015-11-13 17:37:35,935 - octoprint.server.api - INFO - Performing command: sudo shutdown -r now
2015-11-13 17:37:59,765 - octoprint.server - INFO - Starting OctoPrint 1.2.0-dev-705-ga34cbc5-dirty (devel branch)
2015-11-13 17:37:59,771 - octoprint.plugin.core - INFO - Loading plugins from /home/pi/.octoprint/plugins, /home/pi/OctoPrint/src/octoprint/plugins and installed plugin packages...
2015-11-13 17:37:59,966 - octoprint.plugins.discovery - INFO - pybonjour is not installed, Zeroconf Discovery won't be available
2015-11-13 17:37:59,969 - octoprint.plugin.core - INFO - Found 2 plugin(s) providing 2 mixin implementations, 0 hook handlers
2015-11-13 17:37:59,977 - octoprint.filemanager - INFO - Adding backlog items from local to analysis queue
2015-11-13 17:38:01,862 - octoprint.plugin.core - INFO - 2 plugin(s) registered with the system:
|  CuraEngine (bundled) = /home/pi/OctoPrint/src/octoprint/plugins/cura
|  Discovery (bundled) = /home/pi/OctoPrint/src/octoprint/plugins/discovery
2015-11-13 17:38:02,170 - octoprint.server - INFO - Listening on http://0.0.0.0:5000
2015-11-13 17:38:02,172 - octoprint.plugins.discovery - INFO - Registered OctoPrint instance "Kudo3D" for SSDP
2015-11-13 17:38:19,386 - octoprint.server.util.sockjs - INFO - New connection from client: 192.168.0.194
2015-11-13 17:39:52,673 - octoprint.server.api - INFO - Performing command: sudo shutdown -h now
2015-11-13 17:40:16,693 - octoprint.server - INFO - Starting OctoPrint 1.2.0-dev-705-ga34cbc5-dirty (devel branch)
2015-11-13 17:40:16,697 - octoprint.plugin.core - INFO - Loading plugins from /home/pi/.octoprint/plugins, /home/pi/OctoPrint/src/octoprint/plugins and installed plugin packages...
2015-11-13 17:40:16,827 - octoprint.plugins.discovery - INFO - pybonjour is not installed, Zeroconf Discovery won't be available
2015-11-13 17:40:16,829 - octoprint.plugin.core - INFO - Found 2 plugin(s) providing 2 mixin implementations, 0 hook handlers
2015-11-13 17:40:16,833 - octoprint.filemanager - INFO - Adding backlog items from local to analysis queue
2015-11-13 17:40:17,475 - octoprint.plugin.core - INFO - 2 plugin(s) registered with the system:
|  CuraEngine (bundled) = /home/pi/OctoPrint/src/octoprint/plugins/cura
|  Discovery (bundled) = /home/pi/OctoPrint/src/octoprint/plugins/discovery
2015-11-13 17:40:17,826 - octoprint.server - INFO - Listening on http://0.0.0.0:5000
2015-11-13 17:40:17,829 - octoprint.plugins.discovery - INFO - Registered OctoPrint instance "Kudo3D" for SSDP
2015-11-13 18:02:22,447 - octoprint.plugins.cura - INFO - Path to CuraEngine has not been configured yet or does not exist (currently set to None), Cura will not be selectable for slicing
2015-11-13 18:02:22,953 - octoprint.server.util.sockjs - INFO - New connection from client: 192.168.0.194
2015-11-13 18:03:49,281 - octoprint.plugins.cura - INFO - Path to CuraEngine has not been configured yet or does not exist (currently set to None), Cura will not be selectable for slicing
2015-11-13 18:03:49,544 - octoprint.server.util.sockjs - INFO - Client connection closed: 192.168.0.194
2015-11-13 18:03:49,664 - octoprint.server.util.sockjs - INFO - New connection from client: 192.168.0.194
2015-11-13 17:40:15,306 - octoprint.server - INFO - Starting OctoPrint 1.2.0-dev-705-ga34cbc5-dirty (devel branch)
2015-11-13 17:40:15,310 - octoprint.plugin.core - INFO - Loading plugins from /home/pi/.octoprint/plugins, /home/pi/OctoPrint/src/octoprint/plugins and installed plugin packages...
2015-11-13 17:40:15,440 - octoprint.plugins.discovery - INFO - pybonjour is not installed, Zeroconf Discovery won't be available
2015-11-13 17:40:15,442 - octoprint.plugin.core - INFO - Found 2 plugin(s) providing 2 mixin implementations, 0 hook handlers
2015-11-13 17:40:15,446 - octoprint.filemanager - INFO - Adding backlog items from local to analysis queue
2015-11-13 17:40:16,538 - octoprint.plugin.core - INFO - 2 plugin(s) registered with the system:
|  CuraEngine (bundled) = /home/pi/OctoPrint/src/octoprint/plugins/cura
|  Discovery (bundled) = /home/pi/OctoPrint/src/octoprint/plugins/discovery
2015-11-13 17:40:16,729 - octoprint.plugins.discovery - INFO - Registered OctoPrint instance "Kudo3D" for SSDP
2015-11-13 17:40:16,737 - octoprint.server - INFO - Listening on http://0.0.0.0:5000
2015-11-13 18:28:54,062 - octoprint.plugins.cura - INFO - Path to CuraEngine has not been configured yet or does not exist (currently set to None), Cura will not be selectable for slicing
2015-11-13 18:28:54,423 - octoprint.server.util.sockjs - INFO - New connection from client: 192.168.0.194
2015-11-13 18:36:05,154 - octoprint.server.api - INFO - Performing command: sudo shutdown -h now
2015-11-13 18:36:29,837 - octoprint.server - INFO - Starting OctoPrint 1.2.0-dev-705-ga34cbc5-dirty (devel branch)
2015-11-13 18:36:29,842 - octoprint.plugin.core - INFO - Loading plugins from /home/pi/.octoprint/plugins, /home/pi/OctoPrint/src/octoprint/plugins and installed plugin packages...
2015-11-13 18:36:30,042 - octoprint.plugins.discovery - INFO - pybonjour is not installed, Zeroconf Discovery won't be available
2015-11-13 18:36:30,045 - octoprint.plugin.core - INFO - Found 2 plugin(s) providing 2 mixin implementations, 0 hook handlers
2015-11-13 18:36:30,053 - octoprint.filemanager - INFO - Adding backlog items from local to analysis queue
2015-11-13 18:36:31,938 - octoprint.plugin.core - INFO - 2 plugin(s) registered with the system:
|  CuraEngine (bundled) = /home/pi/OctoPrint/src/octoprint/plugins/cura
|  Discovery (bundled) = /home/pi/OctoPrint/src/octoprint/plugins/discovery
2015-11-13 18:36:32,252 - octoprint.server - INFO - Listening on http://0.0.0.0:5000
2015-11-13 18:36:32,253 - octoprint.plugins.discovery - INFO - Registered OctoPrint instance "Kudo3D" for SSDP
2015-11-13 18:43:10,739 - octoprint.plugins.cura - INFO - Path to CuraEngine has not been configured yet or does not exist (currently set to None), Cura will not be selectable for slicing
2015-11-13 18:43:11,101 - octoprint.server.util.sockjs - INFO - New connection from client: 192.168.0.194
2015-11-13 18:47:35,781 - octoprint - ERROR - Exception on /api/job [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/Flask-0.9-py2.7.egg/flask/app.py", line 1687, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/Flask-0.9-py2.7.egg/flask/app.py", line 1360, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python2.7/dist-packages/Flask-0.9-py2.7.egg/flask/app.py", line 1358, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/Flask-0.9-py2.7.egg/flask/app.py", line 1344, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/pi/OctoPrint/src/octoprint/server/util/flask.py", line 214, in decorated_view
    return flask.ext.login.login_required(func)(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/Flask_Login-0.2.2-py2.7.egg/flask_login.py", line 637, in decorated_view
    return func(*args, **kwargs)
  File "/home/pi/OctoPrint/src/octoprint/server/api/job.py", line 38, in controlJob
    printer.start_print()
  File "/home/pi/OctoPrint/src/octoprint/printer/standard.py", line 963, in start_print
    self._comm.startPrint()
  File "/home/pi/OctoPrint/src/octoprint/util/comm.py", line 488, in startPrint
    raise ValueError("No file selected for printing")
ValueError: No file selected for printing
2015-11-13 18:47:44,201 - octoprint - ERROR - Exception on /api/job [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/Flask-0.9-py2.7.egg/flask/app.py", line 1687, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/Flask-0.9-py2.7.egg/flask/app.py", line 1360, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python2.7/dist-packages/Flask-0.9-py2.7.egg/flask/app.py", line 1358, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/Flask-0.9-py2.7.egg/flask/app.py", line 1344, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/pi/OctoPrint/src/octoprint/server/util/flask.py", line 214, in decorated_view
    return flask.ext.login.login_required(func)(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/Flask_Login-0.2.2-py2.7.egg/flask_login.py", line 637, in decorated_view
    return func(*args, **kwargs)
  File "/home/pi/OctoPrint/src/octoprint/server/api/job.py", line 38, in controlJob
    printer.start_print()
  File "/home/pi/OctoPrint/src/octoprint/printer/standard.py", line 963, in start_print
    self._comm.startPrint()
  File "/home/pi/OctoPrint/src/octoprint/util/comm.py", line 488, in startPrint
    raise ValueError("No file selected for printing")
ValueError: No file selected for printing
2015-11-13 18:48:05,163 - octoprint - ERROR - Exception on /api/job [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/Flask-0.9-py2.7.egg/flask/app.py", line 1687, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/Flask-0.9-py2.7.egg/flask/app.py", line 1360, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python2.7/dist-packages/Flask-0.9-py2.7.egg/flask/app.py", line 1358, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/Flask-0.9-py2.7.egg/flask/app.py", line 1344, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/pi/OctoPrint/src/octoprint/server/util/flask.py", line 214, in decorated_view
    return flask.ext.login.login_required(func)(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/Flask_Login-0.2.2-py2.7.egg/flask_login.py", line 637, in decorated_view
    return func(*args, **kwargs)
  File "/home/pi/OctoPrint/src/octoprint/server/api/job.py", line 38, in controlJob
    printer.start_print()
  File "/home/pi/OctoPrint/src/octoprint/printer/standard.py", line 963, in start_print
    self._comm.startPrint()
  File "/home/pi/OctoPrint/src/octoprint/util/comm.py", line 488, in startPrint
    raise ValueError("No file selected for printing")
ValueError: No file selected for printing
2015-11-13 18:48:11,253 - octoprint.server.util.sockjs - INFO - Client connection closed: 192.168.0.194
2015-11-13 18:48:12,618 - octoprint.plugins.cura - INFO - Path to CuraEngine has not been configured yet or does not exist (currently set to None), Cura will not be selectable for slicing
2015-11-13 18:48:12,985 - octoprint.server.util.sockjs - INFO - New connection from client: 192.168.0.194
2015-11-13 18:48:21,549 - octoprint - ERROR - Exception on /api/job [POST]
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/Flask-0.9-py2.7.egg/flask/app.py", line 1687, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/Flask-0.9-py2.7.egg/flask/app.py", line 1360, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python2.7/dist-packages/Flask-0.9-py2.7.egg/flask/app.py", line 1358, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/Flask-0.9-py2.7.egg/flask/app.py", line 1344, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/pi/OctoPrint/src/octoprint/server/util/flask.py", line 214, in decorated_view
    return flask.ext.login.login_required(func)(*args, **kwargs)
  File "/usr/local/lib/python2.7/dist-packages/Flask_Login-0.2.2-py2.7.egg/flask_login.py", line 637, in decorated_view
    return func(*args, **kwargs)
  File "/home/pi/OctoPrint/src/octoprint/server/api/job.py", line 38, in controlJob
    printer.start_print()
  File "/home/pi/OctoPrint/src/octoprint/printer/standard.py", line 963, in start_print
    self._comm.startPrint()
  File "/home/pi/OctoPrint/src/octoprint/util/comm.py", line 488, in startPrint
    raise ValueError("No file selected for printing")
ValueError: No file selected for printing
2015-11-13 18:48:45,295 - octoprint.server.api - INFO - Performing command: sudo shutdown -h now
2015-11-13 18:49:08,869 - octoprint.server - INFO - Starting OctoPrint 1.2.0-dev-705-ga34cbc5-dirty (devel branch)
2015-11-13 18:49:08,874 - octoprint.plugin.core - INFO - Loading plugins from /home/pi/.octoprint/plugins, /home/pi/OctoPrint/src/octoprint/plugins and installed plugin packages...
2015-11-13 18:49:09,071 - octoprint.plugins.discovery - INFO - pybonjour is not installed, Zeroconf Discovery won't be available
2015-11-13 18:49:09,074 - octoprint.plugin.core - INFO - Found 2 plugin(s) providing 2 mixin implementations, 0 hook handlers
2015-11-13 18:49:09,082 - octoprint.filemanager - INFO - Adding backlog items from local to analysis queue
2015-11-13 18:49:10,944 - octoprint.plugin.core - INFO - 2 plugin(s) registered with the system:
|  CuraEngine (bundled) = /home/pi/OctoPrint/src/octoprint/plugins/cura
|  Discovery (bundled) = /home/pi/OctoPrint/src/octoprint/plugins/discovery
2015-11-13 18:49:11,252 - octoprint.server - INFO - Listening on http://0.0.0.0:5000
2015-11-13 18:49:11,255 - octoprint.plugins.discovery - INFO - Registered OctoPrint instance "Kudo3D" for SSDP
2015-11-13 18:51:47,135 - octoprint.plugins.cura - INFO - Path to CuraEngine has not been configured yet or does not exist (currently set to None), Cura will not be selectable for slicing
2015-11-13 18:51:47,537 - octoprint.server.util.sockjs - INFO - New connection from client: 192.168.0.194
2015-11-13 19:04:57,972 - octoprint.server.api - INFO - Performing command: sudo shutdown -h now
2015-11-13 19:05:20,989 - octoprint.server - INFO - Starting OctoPrint 1.2.0-dev-705-ga34cbc5-dirty (devel branch)
2015-11-13 19:05:20,994 - octoprint.plugin.core - INFO - Loading plugins from /home/pi/.octoprint/plugins, /home/pi/OctoPrint/src/octoprint/plugins and installed plugin packages...
2015-11-13 19:05:21,125 - octoprint.plugins.discovery - INFO - pybonjour is not installed, Zeroconf Discovery won't be available
2015-11-13 19:05:21,126 - octoprint.plugin.core - INFO - Found 2 plugin(s) providing 2 mixin implementations, 0 hook handlers
2015-11-13 19:05:21,131 - octoprint.filemanager - INFO - Adding backlog items from local to analysis queue
2015-11-13 19:05:22,307 - octoprint.plugin.core - INFO - 2 plugin(s) registered with the system:
|  CuraEngine (bundled) = /home/pi/OctoPrint/src/octoprint/plugins/cura
|  Discovery (bundled) = /home/pi/OctoPrint/src/octoprint/plugins/discovery
2015-11-13 19:05:22,510 - octoprint.plugins.discovery - INFO - Registered OctoPrint instance "Kudo3D" for SSDP
2015-11-13 19:05:22,511 - octoprint.server - INFO - Listening on http://0.0.0.0:5000
